index.js

```diff
@@ -105,6 +105,8 @@ const builder = new addonBuilder({
     name: 'Strelingo - Dual Language Subtitles',
     description: 'Provides dual subtitles (main + translation) from OpenSubtitles for language learning. <br><a href="https://github.com/Serkali-sudo/strelingo-addon" style="color: #1E90FF;">GitHub</a>',
     resources: ['subtitles'],
+    // Request video hash/size/filename from Stremio for better subtitle matching
+    subtitleExtra: ['videoHash', 'videoSize', 'filename'],
```

Comment:
Hmm thats odd. becuase even without this i remember sdk was returning these values anyways.